### PR TITLE
Update the custom builders section

### PR DESCRIPTION
**Pull request by zware**

### CODE REVIEW

File: buildbots.rst [hunk 1]
```diff
@@ -193,45 +193,31 @@ or by making its parameters - such as a timeout - more robust.
...
-enables issue-specific debugging, you will probably want to test your
-work on one or several buildbots.  Since your work is hosted in a distinct
-repository, you can't trigger builds on the regular builders.  Instead,
-you have to use one of the `custom builders
+When working on a platform-specific issue, you may want to test your changes on
+the buildbot fleet rather than just on Travis and AppVeyor.  To do so, you can
+make use of the `custom builders
 <http://buildbot.python.org/all/waterfall?category=custom.stable&category=custom.unstable>`_.
+These builders track the ``buildbot-custom`` short-lived branch of the
+``python/cpython`` repository, which is only accessible to core developers.
```
Comment (zware) [inline buildbots.rst]: @willingc Only core devs, as this line says.  Is there a better way to phrase this?

Comment (willingc) [inline buildbots.rst]: Thanks @zware. Sorry, I totally missed that line. It's fine as written or perhaps start the section with "Core developers have the option when working on a platform-specific issue...".

**Comment by willingc** [comment]
Thanks @zware :sunny:

**Comment by zware** [comment]
Thanks for merging, @willingc.  I had meant to reword the core-dev bit, but never back got to it.

**Comment by willingc** [comment]
No problem @zware. It was only a suggestion ;-) I'm happy that the content exists :sunflower:

**Pull request merged**